layouts/book-a-demo/single.html

```
@@ -0,0 +1,273 @@
+    </form>
+  </section>
+{{ end }}
+
+{{ define "script" }}
+  {{ if hugo.IsDevelopment }}
+    {{ $newsletterJs := resources.Get "js/newsletter.js" }}
+    <script type="text/javascript" src="{{ $newsletterJs.RelPermalink }}" defer></script>
+    {{ $bookDemoJs := resources.Get "js/bookdemo.js" }}
+    <script type="text/javascript" src="{{ $bookDemoJs.RelPermalink }}" defer></script>
+    {{ $altchaJs := resources.Get "js/altcha/altcha.js" }}
+    <script type="module" type="text/javascript" src="{{ $altchaJs.RelPermalink }}" defer></script>
```

HTMLHint (1.5.0)

[error] Duplicate of attribute name [ type ] was found.

(attr-no-duplication)